src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+
+    private static final int NUMBER_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        //create figure array
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Think about which variables should be local in the method and which should be class-level.